crates/router/src/connector/airwallex.rs

```diff
@@ -868,13 +869,15 @@ impl api::IncomingWebhook for Airwallex {
     fn get_webhook_object_reference_id(
         &self,
         request: &api::IncomingWebhookRequestDetails<'_>,
-    ) -> CustomResult<String, errors::ConnectorError> {
+    ) -> CustomResult<ObjectReferenceId, errors::ConnectorError> {
         let details: airwallex::AirwallexWebhookData = request
             .body
             .parse_struct("airwallexWebhookData")
             .change_context(errors::ConnectorError::WebhookReferenceIdNotFound)?;
 
-        Ok(details.source_id)
+        Ok(ObjectReferenceId::PaymentId(
+            api_models::payments::PaymentIdType::ConnectorTransactionId(details.source_id),
+        ))
```

Comment:
Need logic to detect type of webhook and return appropriate ObjectReferenceId as opposed to always returning PaymentId